string str
---
int32 responseId

